toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/logic/test_xnvme_tests_xnvme_file.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from ..conftest import xnvme_setup_device
+
+
+@pytest.mark.parametrize(
```

Comment:
No, since ``xnvme_parametrize()`` produces backend configurations with a usable device. In this test only the device (file) is relevant, not the backend configuration.